static/app/views/settings/projectIssueGrouping/index.tsx

```diff
@@ -84,6 +89,8 @@ class ProjectIssueGrouping extends AsyncView<Props, State> {
           )}
         </TextBlock>
 
+        <PermissionAlert access={['project:write']} project={project} />
```

Comment:
```suggestion
        <PermissionAlert project={project} />
```
the default `access` in PermissionAlert is `['project:write']`